### DISCUSSION TITLE
Cross join unnest not working

### DISCUSSION THREAD

**Discussion by kishoreduraisam** [prestodb/presto#15591] [Q&A]
I have an array<string> column and I am trying to use cross join unnest to expand the array. But it doesnt work. All it does is remove the brackets []. For example I have an array column which contains something like [123,456,789].  When I do cross join unnest, all I get back is 123,456,789 . It just removes the brackets. I expect it to create a row each for each of the records in that column. 

Interestingly though, I have other array columns in the table which works as expected. My data is in parquet file format. Any help is appreciated.

@findepi  @zhenxiao @rschlussel  @vkorukanti

**Comment by rschlussel** [comment]
it sounds like your array is improperly formatted, so presto thinks it only has a single entry (e.g. what happens if you do SELECT cardinality(my_array) FROM table).

**Reply by kishoreduraisam** [reply]
Thats correct. The array was improperly formatted. thanks for your response.

**Comment by mosabua** [comment]
Just a note for @kishoreduraisam .. @findepi, the creators of Presto, and many other developers are all working on the community version now. It is now called Trino. https://trino.io/blog/2020/12/27/announcing-trino.html
